src/main/kotlin/widgets/LanguageSelection.kt

```diff
@@ -0,0 +1,13 @@
+package widgets
+
+import data.model.Language
+
+class LanguageSelection(language : Language) : DataSelectionInterface {
```

Comment:
This is app specific; thus doesn't belong in the widgets package

Reply:
Got it